### PR TITLE
Add generic data science notebook to JH deployment

### PR DESCRIPTION
**Pull request by LaVLaS**
Adds the generic data science notebook to the list of `additional` jupyter notebooks

### CODE REVIEW

**Comment by LaVLaS** [comment]
/retest

Trying to hit the JH test failure for debuging

**Comment by LaVLaS** [comment]
/retest

Removing WIP so I can hit the JH test failure for debuging

File: jupyterhub/notebook-images/overlays/additional/generic-data-science-notebook-imagestream.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  name: s2i-generic-data-science-notebook
+spec:
+  lookupPolicy:
+    local: true
+  tags:
+  - annotations:
+      openshift.io/imported-from: quay.io/thoth-station/s2i-generic-data-science-notebook
+    from:
+      kind: DockerImage
+      name: quay.io/thoth-station/s2i-generic-data-science-notebook:v0.0.1
```
Comment (harshad16) [inline jupyterhub/notebook-images/overlays/additional/generic-data-science-notebook-imagestream.yaml:19]: can we update this to version `v0.0.2` 
https://quay.io/repository/thoth-station/s2i-generic-data-science-notebook?tab=tags

**Review by harshad16 — COMMENTED** [review]
@LaVLaS  the version https://quay.io/repository/thoth-station/s2i-generic-data-science-notebook?tab=tags v0.0.2 has the jupyterlab-git

File: jupyterhub/notebook-images/overlays/additional/generic-data-science-notebook-imagestream.yaml (hunk 1 above)
Comment (LaVLaS) [reply jupyterhub/notebook-images/overlays/additional/generic-data-science-notebook-imagestream.yaml]: Updated to the `v0.0.2` tag

**Review by harshad16 — COMMENTED** [review]
/lgtm

**Pull request merged**